cdap-api/src/main/java/co/cask/cdap/api/dataset/lib/BaseTimePartition.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright © 2015 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package co.cask.cdap.api.dataset.lib;
+
+/**
+ * Represents a partition of a time partitioned file set.
+ */
+public interface BaseTimePartition extends BasePartition {
```

Comment:
There is `TimePartitionOutput` also, which doesn't have meta, same as the non-Time versions.